src/components/visualizer/visualizerController.ts

```diff
@@ -58,10 +76,17 @@ export default class VisualizerController {
   }
 
   private async startAsync() {
-    this.debuggerHandle = await window.platform.launchProgram(this.exeFile, data => {
+    const handle = await window.platform.launchProgram(this.exeFile, data => {
       //console.log("Console stdout output", data);
       this.onConsoleOutput([data]);
     });
+    if ('err' in handle) {
```

Comment:
Not super familiar with this syntax, but is this essentially checking if the property `err` is present in `handle`?

Reply:
Yes, that's correct